replica/table.cc

```diff
@@ -1470,7 +1470,7 @@ table::sstables_as_snapshot_source() {
             return make_compacting_reader(
                 std::move(reader),
                 gc_clock::now(),
-                [](const dht::decorated_key&) { return api::min_timestamp; },
+                [](const dht::decorated_key&) { return api::max_timestamp; },
```

Comment:
Should be a separate patch with its own description.

Reply:
Okay, will do it as separate patch. And this change leads `incremental_compaction_data_resurrection_test` to failure. We have to fix the test as well.